cdap-watchdog/pom.xml

```diff
@@ -138,6 +143,12 @@
       <artifactId>log4j-over-slf4j</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
```

Comment:
Are there plans to migrate to JUnit 5? @vsethi09 is it the general direction for CDF and what is the timeline? I don't think we should introduce Junit5 just in 1 module unless we have a migration goal

Reply:
Removed